vividus-plugin-azure-storage-account/src/main/java/org/vividus/azure/storage/blob/BlobStorageSteps.java

```diff
@@ -135,7 +139,7 @@ public void downloadBlobToFile(String blobName, String containerName, String sto
             Set<VariableScope> scopes, String variableName) throws IOException
     {
         String tempFilePath = ResourceUtils.createTempFile(baseFileName).toAbsolutePath().toString();
-        createBlobClient(blobName, containerName, storageAccountKey).downloadToFile(tempFilePath);
+        createBlobClient(blobName, containerName, storageAccountKey).downloadToFile(tempFilePath, true);
```

Comment:
I would rather avoid creating the file in the previous line

Reply:
please disregard this comment, now Java always creates an empty file (no easy way to generate the path to temp file without its creation)